src/bar-chart/interfaces.ts

```diff
@@ -30,12 +29,6 @@ export interface BarChartProps<T extends ChartDataTypes>
    */
   horizontalBars?: boolean;
 
-  /**
-   * When set to `true`, adds a visual emphasis on the zero baseline axis.
-   * See the usage guidelines for more details.
-   */
-  emphasizeBaselineAxis?: boolean;
```

Comment:
Moved to the new `CommonMixedChartProps` interface which contains the props from the mixed chart that are also shared by both line and bar charts. Those props are currently `emphasizeBaselineAxis` and `detailPopoverSeriesContent`.